eclipse-che/che-dashboard/che-dashboard_master_rhel_7.6.sh

```diff
@@ -0,0 +1,57 @@
+# ----------------------------------------------------------------------------
+#
+# Package         : che-dashboard
+# VERSION         : master
+# Source repo     : https://github.com/eclipse/che-workspace-loader
+# Tested on       : rhel_7.6
+# Script License  : EPL-2.0
+# Maintainer      : Bivas Das <bivasda1@in.ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+# ----------------------------------------------------------------------------
+# Prerequisites:
+#
+# docker version supporting multistage builds must be installed and running.
+#
+# The internal resgistry should be set as insecure registry in docker daemon settings.
+#
+# oc client version 4.18 or above must be installed.
+# Deployment tested on Openshift 4.3 setup
+#
+# ----------------------------------------------------------------------------
+
+VERSION=master
+
+set -e
+yum update -y
+yum install git -y

```

Comment:
@ghatwala I have a similar question regarding docker as well. Docker is also not installed by default on RHEL. So irrespective of docker or podman, should the buildscript also try to install these packages? Since we already mention that in the Prerequisites section that docker/podman should be installed on the system.

Reply:
@kumarvikas2020  - suggest you checking other builds scripts in this repo . Just mentioning in the pre-requisite section wont help here at all . All prerequisites to run your build script should be present/installed in the script it self .  All builds  scripts are written in this repo with the assumption that we run them in a clean environment.

Reply:
@ghatwala Thanks for the explanation. I have scanned the whole repository. Almost all the scripts which are using tools like docker, oc, kubectl, etc are not trying to install these tools as dependencies, but are only mentioned in the Prerequisites. So we also took the similar approach.
For the scripts in this PR and for the future scripts, we will take care of this as well.

Reply:
@ghatwala 
The required changes have been done.